trk/src/Navigation/AppNavigator.js

```diff
@@ -65,6 +65,13 @@ function ProfileStack() {
         name="Set"
         component={SetDetail}
         options={{ title: 'Climb Performance', headerBackTitle: 'Profile', headerTitleAlign: 'center' }} />
+
+      <Stack.Screen
+        name="ClimbInputData"
+        component={ClimbInputData}
+        options={{ title: 'Climb Data', headerBackTitle: 'Back', headerTitleAlign: 'center' }}
+      />
+
```

Comment:
I'm not actually creating a new file. Instead, I decided that the Setter Profile's Set Detail should navigate back to ClimbInputData ('CREATE CLIMB FORM') with some tweaks in the form's UI, instead of creating a new file. That way, we can reuse components instead of re-creating new ones with a lot of redundant code.